build-tools/packages/build-cli/src/commands/check/latestVersions.ts

```diff
@@ -62,21 +62,21 @@ export default class LatestVersionsCommand extends BaseCommand<typeof LatestVers
 					this.log(
 						`Version ${versionInput.version} is the latest version for major version ${majorVersion}`,
 					);
+					this.log(`##vso[task.setvariable variable=shouldDeploy;isOutput=true]true`);
 					return;
 				}
 
 				// If versions do not match on first major version encounter, then the input version is not the latest
-				this.error(
+				this.log(
 					`skipping deployment stage. input version ${versionInput.version} does not match the latest version ${v.version}`,
-					{ exit: 1 },
 				);
+				this.log(`##vso[task.setvariable variable=shouldDeploy;isOutput=true]false`);
+				return;
```

Comment:
I'm conflicted about this approach. Maybe formatting the messages as warnings using the info here would be enough? https://learn.microsoft.com/en-us/azure/devops/pipelines/scripts/logging-commands?view=azure-devops&tabs=bash#formatting-commands

otherwise where how can we tell if the step succeeded?